judge/chroot-startstop.sh.in

```diff
@@ -35,6 +35,19 @@ SUBDIRMOUNTS="etc usr lib lib64 bin"
 # Location of the pre-built chroot tree and where to bind mount from:
 CHROOTORIGINAL="@judgehost_chrootdir@"
 
+dj_umount() {
+	set +e
+	if ! sudo -n umount "$1" < /dev/null; then
+		>&2 echo "umount '$1' didn't succeed, trying harder"
+		if ! sudo -n umount -f -vvv "$1" < /dev/null; then
+			>&2 echo "umount '$1' failed twice"
+			>&2 lsof +c 15 +D "$1"
+			exit -1
```

Comment:
Shell exit statuses should be in the range 0-255, so I would suggest to use either 1 or 255.